samples/healthcheck/pom.xml

```diff
@@ -39,6 +39,11 @@
                   <cmd>curl -f http://localhost/ || exit 1</cmd>
                 </healthCheck>
               </build>
+              <run>
+                <wait>
+                  <healthy>true</healthy>
```

Comment:
It's more a complaint against Maven ;-). No, attributes are not possible at all for Maven configurations, no empty tags, too. 

So its fine for me as it is now, except when we would find something 'useful' to put as value (like a timeout, but thats also available otherwise). But no worries here ...